src/helper/helper.ts

```diff
@@ -2,3 +2,7 @@ import moment from 'moment'
 
 export const getDateFormat = (date: string) => moment(date).format('DD-MM-YYYY')
 export const getDateNow = () => moment().format('DD-MM-YYYY')
+
+export const getStatusValue = (data: string[]) => {
+  return data.length ? data.map((value) => `status=${value}`).join('&') : ''
```

Comment:
what is the main purpose of this function, looks really strange

Reply:
I create a method in the helper, where I throw the current Status and returns a string, this is so as not to write such a request to requestOrdersInfo in the url

Reply:
> what is the main purpose of this function, looks really strange

I create a method in the helper, where I throw the current Status and returns a string, this is so as not to write such a request to requestOrdersInfo in the url